config.py

```diff
@@ -0,0 +1,27 @@
+"""Flask config."""
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+class Config:
+    """Base config."""
+
+    FLASK_ENV = 'development'
+    TESTING = True
```

Comment:
The base class set TESTING=True, then one child class overrode with TESTING=FALSE and another one overrode with TESTING=True (no effect). I think that's a bit confusing.

What if you choose reasonable prod defaults and put them in the base class. E.g. Config sets TESTING=False, prod doesn't override, but DevConfig overrides with TESTING=True